pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -4379,6 +4408,138 @@ def DisposalDestinationDeliveriesRule(model, k, t):
     return model
 
 
+def scale_model(model, scaling_factor=None):
+
+    if scaling_factor is None:
+        scaling_factor = 1000000
+
+    model.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+    # Scaling variables
+    model.scaling_factor[model.v_Z] = 1 / scaling_factor
```

Comment:
Discussed with Andres: the automated tests will indicate if vars / constraints are not scaled. That's preferred because otherwise variables might be incorrectly scaled.